io-engine/src/bin/io-engine-client/v1/rebuild_cli.rs

```diff
@@ -542,7 +562,7 @@ async fn progress(
 
     let response = ctx
         .client
-        .get_rebuild_progress(rpc::RebuildProgressRequest {
+        .get_rebuild_progress(v0::RebuildProgressRequest {
```

Comment:
why are we using v0 in v1?

Reply:
v1 has no "get_rebuild_progress" API.
Ok, I will change to use rebuild_stats().